docs/modules/ROOT/pages/user-code-namespaces.adoc

```diff
@@ -55,6 +55,8 @@ include::ROOT:example$/user-code-namespace-azure.yaml[]
 <1> The bucket URI in which to store backup data
 <2> Name of the secret with credentials for accessing the Bucket URI.
 
+NOTE: Install public buckets using the `remoteURLs` field over HTTP.
```

Comment:
UCN does not implement `remoteURLs` field for now

Reply:
Do we have a plan to support it?